Update Set-CsOnlineLisPort.md

danny-levin opened the pull request:
Added 2 examples

opbld32 commented:
Docs Build status updates of commit _[43f19d8](https://github.com/danny-levin/office-docs-powershell/commits/43f19d8bff44bd91a3ebeb8c49c4681db5085f98)_: 

:clock10: **Full build**: the average full build time is _17 min(s) 33 sec(s)_, this is based on the last _1000_ build(s) for this repository.

opbld32 commented:
Docs Build status updates of commit _[43f19d8](https://github.com/danny-levin/office-docs-powershell/commits/43f19d8bff44bd91a3ebeb8c49c4681db5085f98)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/Set-CsOnlineLisPort.md](https://github.com/danny-levin/office-docs-powershell/blob/patch-6/skype/skype-ps/skype/Set-CsOnlineLisPort.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinelisport?view=skype-ps&branch=pr-en-us-6086) (skype-ps) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C6%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202007061739595965-6086%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Jd86s9Au%2BNrjGSnAhdjpqjHQefJqE0iUJrMcIPI4mIM%3D&st=2020-07-06T17%3A57%3A36Z&se=2020-08-06T18%3A02%3A36Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

dariomws commented on skype/skype-ps/skype/Set-CsOnlineLisPort.md:
````diff
@@ -36,6 +36,18 @@ Set-CsOnlineLisPort -PortID 12174 -ChassisID 0B-23-CD-16-AA-CC -Description "Lis
+Set-CsOnlineLisPort -PortID 0A-25-55-AB-CD-FF -ChassisID 0B-23-CD-16-AA-CC -Description "LisPort 0A-25-55-AB-CD-FF" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
+```
+
+Example 2 creayes the association between port "0A-25-55-AB-CD-FF" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".
````
```suggestion
Example 2 creates the association between port "0A-25-55-AB-CD-FF" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".
```

dariomws commented on skype/skype-ps/skype/Set-CsOnlineLisPort.md:
````diff
@@ -36,6 +36,18 @@ Set-CsOnlineLisPort -PortID 12174 -ChassisID 0B-23-CD-16-AA-CC -Description "Lis
+Set-CsOnlineLisPort -PortID 12174 -ChassisID 55123 -Description "LisPort 12174" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
+```
+
+Example 3 creayes the association between port "12174" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee". (Note: in this example, ChassisID sub-type  is InterfaceName) 
````
```suggestion
Example 3 creates the association between port "12174" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee". (Note: in this example, ChassisID sub-type  is InterfaceName) 
```

dariomws commented on skype/skype-ps/skype/Set-CsOnlineLisPort.md:
````diff
@@ -36,6 +36,18 @@ Set-CsOnlineLisPort -PortID 12174 -ChassisID 0B-23-CD-16-AA-CC -Description "Lis
 Example 1 creayes the association between port "12174" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".
+### -------------------------- Example 2 --------------------------
+```
+Set-CsOnlineLisPort -PortID 0A-25-55-AB-CD-FF -ChassisID 0B-23-CD-16-AA-CC -Description "LisPort 0A-25-55-AB-CD-FF" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
````
```suggestion
PS C:\> Set-CsOnlineLisPort -PortID 0A-25-55-AB-CD-FF -ChassisID 0B-23-CD-16-AA-CC -Description "LisPort 0A-25-55-AB-CD-FF" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
```

dariomws commented on skype/skype-ps/skype/Set-CsOnlineLisPort.md:
````diff
@@ -36,6 +36,18 @@ Set-CsOnlineLisPort -PortID 12174 -ChassisID 0B-23-CD-16-AA-CC -Description "Lis
+Example 2 creayes the association between port "0A-25-55-AB-CD-FF" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".
+### -------------------------- Example 3 --------------------------
+```
+Set-CsOnlineLisPort -PortID 12174 -ChassisID 55123 -Description "LisPort 12174" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
````
```suggestion
PS C:\> Set-CsOnlineLisPort -PortID 12174 -ChassisID 55123 -Description "LisPort 12174" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
```

get-itips commented:
agree with @dariomws

danny-levin commented:
Suggested change is the same as current text, hence I am not clear what change is requested

From: Dario Woitasen <notifications@github.com>
Sent: Thursday, July 9, 2020 5:49 PM
To: MicrosoftDocs/office-docs-powershell <office-docs-powershell@noreply.github.com>
Cc: Danny Levin <dannyle@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [MicrosoftDocs/office-docs-powershell] Update Set-CsOnlineLisPort.md (#6086)


@dariomws requested changes on this pull request.

________________________________

In skype/skype-ps/skype/Set-CsOnlineLisPort.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Foffice-docs-powershell%2Fpull%2F6086%23discussion_r452267973&data=02%7C01%7Cdannyle%40microsoft.com%7Cf967657279544786525308d8241728bb%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637299029175336470&sdata=0Xsw%2FUoE%2BCQuZutqB63vOjOn6biDyRa%2FVJRs42yJpig%3D&reserved=0>:

> @@ -36,6 +36,18 @@ Set-CsOnlineLisPort -PortID 12174 -ChassisID 0B-23-CD-16-AA-CC -Description "Lis



 ```







 Example 1 creayes the association between port "12174" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".



+### -------------------------- Example 2 --------------------------



+```



+Set-CsOnlineLisPort -PortID 0A-25-55-AB-CD-FF -ChassisID 0B-23-CD-16-AA-CC -Description "LisPort 0A-25-55-AB-CD-FF" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee


⬇️ Suggested change

-Set-CsOnlineLisPort -PortID 0A-25-55-AB-CD-FF -ChassisID 0B-23-CD-16-AA-CC -Description "LisPort 0A-25-55-AB-CD-FF" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee



+PS C:\> Set-CsOnlineLisPort -PortID 0A-25-55-AB-CD-FF -ChassisID 0B-23-CD-16-AA-CC -Description "LisPort 0A-25-55-AB-CD-FF" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee



________________________________

In skype/skype-ps/skype/Set-CsOnlineLisPort.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Foffice-docs-powershell%2Fpull%2F6086%23discussion_r452268145&data=02%7C01%7Cdannyle%40microsoft.com%7Cf967657279544786525308d8241728bb%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637299029175336470&sdata=3Yaa74DRgqgZkIE6Voclg8laY8l1WeKWCCCTZvUYNB8%3D&reserved=0>:

> @@ -36,6 +36,18 @@ Set-CsOnlineLisPort -PortID 12174 -ChassisID 0B-23-CD-16-AA-CC -Description "Lis



 ```







 Example 1 creayes the association between port "12174" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".



+### -------------------------- Example 2 --------------------------



+```



+Set-CsOnlineLisPort -PortID 0A-25-55-AB-CD-FF -ChassisID 0B-23-CD-16-AA-CC -Description "LisPort 0A-25-55-AB-CD-FF" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee



+```



+



+Example 2 creayes the association between port "0A-25-55-AB-CD-FF" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".



+### -------------------------- Example 3 --------------------------



+```



+Set-CsOnlineLisPort -PortID 12174 -ChassisID 55123 -Description "LisPort 12174" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee


⬇️ Suggested change

-Set-CsOnlineLisPort -PortID 12174 -ChassisID 55123 -Description "LisPort 12174" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee



+PS C:\> Set-CsOnlineLisPort -PortID 12174 -ChassisID 55123 -Description "LisPort 12174" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee



—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Foffice-docs-powershell%2Fpull%2F6086%23pullrequestreview-445682637&data=02%7C01%7Cdannyle%40microsoft.com%7Cf967657279544786525308d8241728bb%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637299029175346464&sdata=1i5EtHR3invF6sjoVgwsc019OJKzAAGyWoRZE8odtmU%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAJLYUC6I6RJIWH6A22KPFH3R2XKEHANCNFSM4OR2KNZA&data=02%7C01%7Cdannyle%40microsoft.com%7Cf967657279544786525308d8241728bb%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637299029175346464&sdata=KRGwa6rCqVm3SDOCEgQCr4vmNtFQMAck0sy2YkiNGC0%3D&reserved=0>.

dariomws commented:
Hi @danny-levin, thank you for your help to improve docs.microsoft.com.

The most important change I'm suggesting is to replace "creayes" with "creates". I think it's a typo in the first example.

Also, I'm suggesting to add "PS C:\> " to each command. You should have the option to commit these updates.

opbld33 commented:
Docs Build status updates of commit _[0133150](https://github.com/danny-levin/office-docs-powershell/commits/0133150d80126278a6e89492341edd15900ed9d9)_: 

:clock10: **Full build**: the average full build time is _17 min(s) 36 sec(s)_, this is based on the last _1000_ build(s) for this repository.

danny-levin commented:
Thanks for catching it.  committed requested changes

dariomws commented:
@danny-levin Thanks again, for taking out some time to create this pull request. I appreciate and encourage you to do the same in the future also.

Please, don´t close it as it needs to be merged to appear in docs.microsoft.com. Thanks!

danny-levin commented:
Thanks a lot for catching it.
I added 2 examples for using various sub types.
I cut & paste the first example and changed the types.
I assume that the typo was before but I didn’t notice…

BTW the 2 examples that I added are not shown here  https://docs.microsoft.com/en-us/powershell/module/skype/set-csonlinelisport?view=skype-ps
When will they be published?


From: Dario Woitasen <notifications@github.com>
Sent: Thursday, July 9, 2020 6:01 PM
To: MicrosoftDocs/office-docs-powershell <office-docs-powershell@noreply.github.com>
Cc: Danny Levin <dannyle@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [MicrosoftDocs/office-docs-powershell] Update Set-CsOnlineLisPort.md (#6086)


Hi @danny-levin<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdanny-levin&data=02%7C01%7Cdannyle%40microsoft.com%7Cc540de0c48ca44a0456b08d82418f625%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637299036921998332&sdata=ZmyJ%2FTEQus54xxkSfcB5FMndFo08cZZAaChvrBHX7%2Fw%3D&reserved=0>, thank you for your help to improve docs.microsoft.com.

The most important change I'm suggesting is to replace "creayes" with "creates". I think it's a typo in the first example.

Also, I'm suggesting to add "PS C:> " to each command. You should have the option to commit these updates.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Foffice-docs-powershell%2Fpull%2F6086%23issuecomment-656180427&data=02%7C01%7Cdannyle%40microsoft.com%7Cc540de0c48ca44a0456b08d82418f625%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637299036922008326&sdata=XCL%2B3Og5MJJurJa50y7ZqfkbGSfEUBVeUUvlauboExs%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAJLYUC2DC7X2HMQ6V5RLDDLR2XLUTANCNFSM4OR2KNZA&data=02%7C01%7Cdannyle%40microsoft.com%7Cc540de0c48ca44a0456b08d82418f625%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637299036922008326&sdata=cOF%2Bjbfmdgi30q%2BO6gVYOAp4KCbwL%2Fqi6Gh4Lxz4OKU%3D&reserved=0>.

danny-levin commented:
Thanks for your help😊 – I am new to editing exiting docs.
What do I need to do to complete this process?

From: Dario Woitasen <notifications@github.com>
Sent: Thursday, July 9, 2020 6:15 PM
To: MicrosoftDocs/office-docs-powershell <office-docs-powershell@noreply.github.com>
Cc: Danny Levin <dannyle@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [MicrosoftDocs/office-docs-powershell] Update Set-CsOnlineLisPort.md (#6086)


@danny-levin<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdanny-levin&data=02%7C01%7Cdannyle%40microsoft.com%7C5ede4a567da3490be7e408d8241ad199%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637299044891061660&sdata=SyjDfWKQXGHo5Yh6x87fZ6Oc140zKcNsOwgimjz0kzQ%3D&reserved=0> Thanks again, for taking out some time to create this pull request. I appreciate and encourage you to do the same in the future also.

Please, don´t close it as it needs to be merged to appear in docs.microsoft.com. Thanks!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Foffice-docs-powershell%2Fpull%2F6086%23issuecomment-656187903&data=02%7C01%7Cdannyle%40microsoft.com%7C5ede4a567da3490be7e408d8241ad199%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637299044891061660&sdata=yI%2F%2FZ5L5dvs5hmG6iqf5cXlzYHc00wZYhJLWjVNBsrQ%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAJLYUCY4KDV2DBXHNKB756TR2XNGPANCNFSM4OR2KNZA&data=02%7C01%7Cdannyle%40microsoft.com%7C5ede4a567da3490be7e408d8241ad199%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637299044891071656&sdata=wdOwpH%2Fmy9lkFvB0CFPDU7TdiW8yvs5vryVdcvHKgd4%3D&reserved=0>.

dariomws commented:
@danny-levin you're welcome. When the PR is merged, you'll see the updates in docs.microsoft.com after a few days.

opbld33 commented:
Docs Build status updates of commit _[0133150](https://github.com/danny-levin/office-docs-powershell/commits/0133150d80126278a6e89492341edd15900ed9d9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/Set-CsOnlineLisPort.md](https://github.com/danny-levin/office-docs-powershell/blob/patch-6/skype/skype-ps/skype/Set-CsOnlineLisPort.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinelisport?view=skype-ps&branch=pr-en-us-6086) (skype-ps) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C9%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202007091507255399-6086%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=owNAk9F4aA%2F2gJAUOGwkUoGShgicflafitf4TUlb6go%3D&st=2020-07-09T15%3A23%3A33Z&se=2020-08-09T15%3A28%3A33Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld30 commented:
Docs Build status updates of commit _[02be161](https://github.com/danny-levin/office-docs-powershell/commits/02be16171016af8085e500ba69e620345084f8d3)_: 

:clock10: **Full build**: the average full build time is _17 min(s) 36 sec(s)_, this is based on the last _1000_ build(s) for this repository.

opbld30 commented:
Docs Build status updates of commit _[02be161](https://github.com/danny-levin/office-docs-powershell/commits/02be16171016af8085e500ba69e620345084f8d3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/Set-CsOnlineLisPort.md](https://github.com/danny-levin/office-docs-powershell/blob/patch-6/skype/skype-ps/skype/Set-CsOnlineLisPort.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinelisport?view=skype-ps&branch=pr-en-us-6086) (skype-ps) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C9%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202007091507477457-6086%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=J3qRZMwBCMGFCau2jd%2Bhb%2BFKyGrUB3o9EMNjJUMTuVY%3D&st=2020-07-09T15%3A43%3A51Z&se=2020-08-09T15%3A48%3A51Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

yogkumgit commented:
@chrisda Please Merge this PR. Thanks

opbld32 commented:
Docs Build status updates of commit _[52d1191](https://github.com/danny-levin/office-docs-powershell/commits/52d1191d32894bf3e33bea19ee0fa57421c8a270)_: 

:clock10: **Full build**: the average full build time is _17 min(s) 38 sec(s)_, this is based on the last _1000_ build(s) for this repository.

opbld32 commented:
Docs Build status updates of commit _[52d1191](https://github.com/danny-levin/office-docs-powershell/commits/52d1191d32894bf3e33bea19ee0fa57421c8a270)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/Set-CsOnlineLisPort.md](https://github.com/danny-levin/office-docs-powershell/blob/patch-6/skype/skype-ps/skype/Set-CsOnlineLisPort.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinelisport?view=skype-ps&branch=pr-en-us-6086) (skype-ps) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C10%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202007101710428946-6086%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yOW4MhftwNkapSsZJurj%2BiBTeKhWVuxzsOsfyPlcQEo%3D&st=2020-07-10T17%3A27%3A46Z&se=2020-08-10T17%3A32%3A46Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
